paper/paper.md

```diff
@@ -0,0 +1,86 @@
+
+Capacity expansion planning is used to compute cost-optimal energy system designs under given sets of constraints. The resulting cost-optimal energy system design can be used to inform policy decisions that incentivize the industry to invest in this design [@Johnston:2013]. Similarly, cost-optimal energy system designs can be used by companies for their investment strategies.
+
+Aspects of the energy system design that capacity expansion planning aims to answer are what the optimal technology mix is in regards of location, time, and installed generation, conversion, storage, and transmission capacities. The design optimization is done while using an integrated dispatch formulation to ensure that supply can equal demand at all nodes and time steps. The model determines the costs, emissions, power generation, energy storage, and power flows based on the installed capacities.
+
+Capacity expansion planning is formulated as mathematical optimization problem. Like any optimization problem, capacity expansion planning has certain degrees of freedom, consists of constraints, and consists of an objective function that is minimized: Typical degrees of freedom (sometimes called decision variables) are installed capacities, power generation, energy storage, and power flows. We divide constraints into internal and external constraints: The internal constraints ensure that the model is physically consistent in itself and following the rules of thermodynamics, e.g. energy balances ensure energy conservation over time. External constraints restrict the solution space to external conditions like costs, demands, available energy resources as well as other political, environmental, economical, or technological constraints. The objective function to determine cost-optimal investment can be total system cost, the net present value, or another business-oriented cost measure.
+
+Dispatch planning formulations are very similar to capacity expansion formulations. However, the installed capacity is no degree of freedom, but introduced as an external constraint.
+
+# Package features
+
+The model class of ``CapacityExpansion`` is capacity expansion planning and generation and transmission capacity expansion planning are combined. The model is setup as a linear optimization model that models energy systems based on the provided input data. Multiple energy carriers can be modeled, which makes the software suitable for research of sector coupling technologies. Technologies can be defined that belong to dispathable or non-dispathable generation, conversion, storage, transmission, or demand. The decision variables of the model are investment and dispatch and the total system costs are minimized.
```

Comment:
Something is missing in the first sentence, gramatically.

Reply:
Would:
`The model class of ``CapacityExpansion`` is capacity expansion planning and it combines generation and transmission capacity expansion planning. ` make more sense?